client-frameworks-support/client-support-ui5/src/ui5-support-lib.js

```diff
@@ -0,0 +1,107 @@
+sap.ui.define(['sap/ui/model/json/JSONModel', '@luigi-project/client/luigi-client'], function(JSONModel, LuigiClient) {
```

Comment:
How do you imagine, that this resource is consumed by applications? Also as an ES module? Or as a JS source? As mentioned above - we need to maybe explain both? And we have to align the namespaces...